plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -1200,15 +1177,6 @@ public static function clean_theme_cache() {
 		delete_transient( 'wc_system_status_theme_info' );
 	}
 
-	/**
-	 * Clear the system status plugin caches
-	 */
-	public static function clean_plugin_cache() {
-		delete_transient( 'wc_system_status_active_plugins' );
-		delete_transient( 'wc_system_status_inactive_plugins' );
-		delete_transient( 'wc_system_status_dropins_mu_plugins' );
-	}
```

Comment:
Public methods in existing classes can't be removed (unless the class is in the `Automattic\WooCommerce\Internal` namespace or a subnamespace of that one) in order to keep backwards compatibility.